addons/website/views/snippets/s_media_list.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
 
 <template name="Media List" id="s_media_list">
-    <section class="s_media_list bg-200 pt32 pb32" data-vcss="001">
+    <section class="s_media_list pt32 pb32 o_cc o_cc2" data-vcss="001">
```

Comment:
This needs a theme PR, some themes override all the snippets you modify to change that bg-200 into something else, you have to adapt those.

Reply:
I have pushed a new PR here https://github.com/odoo/design-themes/pull/15